server/odc-core/src/main/resources/i18n/ErrorMessages.properties

```diff
@@ -142,6 +142,7 @@ com.oceanbase.odc.ErrorCodes.ObCreatePlRunPackageFailed=Failed to create the PL
 com.oceanbase.odc.ErrorCodes.ObGetExecuteDetailFailed=Failed to fetch the execution details. Details: {0}.
 com.oceanbase.odc.ErrorCodes.ObGetPlanExplainFailed=Failed to fetch the execution plan due to possible syntax error or invalid DDL statement. Details: {0}.
 com.oceanbase.odc.ErrorCodes.ObGetPlanExplainEmpty=Unable to fetch the execution plan due to possible syntax error, invalid DDL statement, or unsupported command. Details: {0}.
+com.oceanbase.odc.ErrorCodes.IntelligentAnalysisOfExecutionPlanFailed=SQL execution plan intelligent analysis failed, error details: {0}.
```

Comment:
Failed to perform intelligent analysis of the SQL execution plan. Details: {0}.

Reply:
ok～